atc/creds/vault/vault_test.go

```diff
@@ -147,5 +147,30 @@ var _ = Describe("Vault", func() {
 				Expect(err).To(BeNil())
 			})
 		})
+
+		Context("allowRootPath", func() {
+			JustBeforeEach(func() {
```

Comment:
nit: `JustBeforeEach` should only really be used for performing the effect under test, and there should only be one per context; otherwise just use `BeforeEach` for setup

Reply:
@vito I used `JustBeforeEach` here because the original test code initially used `JustBeforeEach` at line#39. OK, let me change all to `BeforeEach`.